docs/app/visualization/QuickstartCreateDashboard.mdx

```diff
@@ -4,7 +4,7 @@ title: Create a Dashboard
 displayed_sidebar: webUiSidebar
 ---
 
-import { DownOutlined, SaveOutlined, UserOutlined } from '@ant-design/icons';
+import { DownOutlined, SaveOutlined, UserOutlined, AreaChartOutlined } from '@ant-design/icons';
```

Comment:
nothing wrong with this line - but more just to know - in case this took a while to find, can usually find the icon in the api-server repo by looking for the appropriate component.